_posts/en/newsletters/2023-03-15-newsletter.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Bitcoin Optech Newsletter #242'
+permalink: /en/newsletters/2023/03/15/
+name: 2023-03-15-newsletter
+slug: 2023-03-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Service bit for Utreexo:** Calvin Kim [posted][kim utreexo] to the
+  Bitcoin-Dev mailing list that software currently designed for
+  experimentation on signet and testnet will be using P2P protocol
+  service bit 24.  The experimental software provides support for
+  [Utreexo][topic utreexo], a protocol for allowing full verification of
+  transaction by software which doesn't store a copy of the UTXO set,
+  saving up to 5 GB of disk space compared to a modern Bitcoin Core full
+  node without any reduction in security.  A Utreexo node needs to
+  receive extra data when it receives a transaction (or a block full of
+  transactions), so the service bit will help a node find peers capable
+  of providing the extra data.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.16.0-beta.rc2][] is a release candidate for a new major
+  version of this popular LN implementation.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25740][] allows a node using [assumeUTXO][topic
+  assumeutxo] to download all blocks before the assumed-valid block,
+  verify each of them (including all of their transactions), and build a
+  UTXO set (chainstate) as of that block.  If that chainstate is equal
+  to the assumeUTXO state used when the node was first started, then the
```

Comment:
If Im understanding the context, "the assumeUTXO state used" seems too opaque to me. Maybe something about matching the value or hash in the source code.

Reply:
I slightly rephrased this but I don't think it fully addresses your concern.  My take is that I think being more specific about the current plans for the (temporary) source of truth distracts from describing this change and it may introduce confusion.   By mentioning a hash is hardcoded, it seems to imply that trust is involved, but the whole point of background validating the assumeUTXO state is that trust isn't required in the long term.

@bitschmidty as always, if you feel strongly, please feel free to edit before publication.

Reply:
ACK